Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphSchemaBuilder.php

```diff
@@ -34,18 +35,18 @@ public function buildSchema(AbstractSchemaManager $schemaManager): Schema
 
     private function createNodeTable(): Table
     {
-        $table = new Table($this->contentGraphTableNames->node(), [
+        $table = self::createTable($this->contentGraphTableNames->node(), [
```

Comment:
the `Table` constructor has a `@throws` annotation annoyingly. I created a little static helper method to catch and rethrow that with a more meaningful message

Reply:
well they could simply use static analysis to assert that the name should not be an empty string ^^ ... these static helpers definitely obscure our code a little :(

Reply:
right.. or throw an `\InvalidArgumentException` in that case not a `Doctrine\DBAL\Exception` – that's bad style. Anyways